app/assets/stylesheets/styles/components/crops/components/tags-input.scss

```diff
@@ -0,0 +1,391 @@
+    }
+  }
+
+  &.ti-input-lg {
+
+    .input-group-lg {
+
+      .keywords {
+        min-height: 46px;
+      }
+    }
+
+    .tags {
+
+      .input-group-lg {
+
+        .keywords {
+
+          .tags {
+            border-radius: 0;
+          }
+        }
+      }
+
+      .tag-item {
+
+        .input-group-lg {
+
+          .keywords {
+
+            .tags {
+
+              .tag-item {
+                height: 38px;
+                line-height: 37px;
+                font-size: 18px;
+                border-radius: 0;
+              }
+            }
+          }
+        }
+
+        .remove-button {
+
+          .input-group-lg {
+
+            .keywords {
+
+              .tags {
+
+                .tag-item {
+
+                  .remove-button {
+                    font-size: 20;
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+
+      .input {
+
+        .input-group-lg {
+
+          .keywords {
+
+            .tags {
+
+              .input {
+                height: 38px;
+                font-size: 18px;
+              }
+            }
+          }
+        }
+      }
+    }
+
+    &.ti-input-sm {
+
+      .input-group-sm {
+
+        .keywords {
+          min-height: 30;
+        }
+      }
+
+      .tags {
+
+        .input-group-sm {
+
+          .keywords {
+
+            .tags {
+              border-radius: 0;
+            }
+          }
+        }
+
+        .tag-item {
+
+          .input-group-sm {
+
+            .keywords {
+
+              .tags {
+
+                .tag-item {
+                  height: 22px;
+                  line-height: 21px;
+                  font-size: 12px;
+                  border-radius: 0;
+                }
+              }
+            }
+          }
+
+          .remove-button {
+
+            .input-group-sm {
+
+              .keywords {
+
+                .tags {
+
+                  .tag-item {
+
+                    .remove-item {
+
+                      font-size: 16px;
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+
+        .input {
+
+          .input-group-sm {
+
+            .keywords {
+
+              .tags {
+
+                .input {
+                  height: 22px;
+                  font-size: 12px;
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
+
+.keywords {
+
+  .tags {
+
+    .tag-item {
+
+      &.selected {
+        background: linear-gradient(to bottom, lighten($of-red, 20%) 0%,
+          lighten($of-red, 30%) 45%, lighten($of-red, 40%) 100%);
+      }
+
+      .remove-button {
+        margin: 0;
+        margin-left: 5px;
+        padding: 0;
+        border: none;
+        background: none;
+        cursor: pointer;
+        vertical-align: middle;
+        font-weight: bold;
+        font-size: 16px;
+        color: $dark-gray;
+      }
+
+      .remove-button:active {
```

Comment:
@TanSA05 how many `.remove-button`s are there? Is there just one `.remove-button` in the code? Then we don't need to nest it inside of `.tags` or `.tag-item`. 

If there's just one `.remove-button` in `.tags`. then we just need to nest it inside of `.tags`.